src/features/game/scenes/bindings/Bindings.ts

```diff
@@ -91,12 +95,32 @@ class Bindings extends Phaser.Scene {
   private getBindings() {
     return [
       {
-        key: 'Esc',
+        key: 'ESC',
```

Comment:
Was there any particular reason for this change?

Reply:
I think it is just an accidental change.